cmd/signals.go

```diff
@@ -38,10 +37,9 @@ func handleSignals() {
 		}
 
 		if success {
-			os.Exit(0)
+			minioExitFunc(0)
```

Comment:
Do we also want to block on regular exit?

Seems like that would make it rather hard to shut it down.

Reply:
yes, you are right, I will avoid regular exit